Library/Homebrew/livecheck/strategy/github_latest.rb

```diff
@@ -48,7 +43,10 @@ class GithubLatest
 
         # The default regex used to identify a version from a tag when a regex
         # isn't provided.
-        DEFAULT_REGEX = %r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i.freeze
+        DEFAULT_REGEX = /v?(\d+(?:\.\d+)+)/i.freeze
+
+        # Keys in the release JSON that could contain the version.
+        VERSION_KEYS = ["name", "tag_name"].freeze
```

Comment:
```suggestion
```

Out of the ~640 existing `GithubLatest` checks, only ~8 match the version from something other than the tag name. We should simply check the `tag_name` by default and checks that need to match a different field can easily use a `strategy` block.

Matching the `name` by default doesn't provide any additional value (it can be problematic in some instances) and we especially don't want to check it before the `tag_name`. The idea is that tags are more likely to have strict formatting and avoid extraneous text, whereas release titles can vary and sometimes contain extraneous text (like even another version).

Only checking the `tag_name` also means that the default behavior of `GithubLatest` will be more in line with the `Git` strategy, making it easier to move from one to the other (excluding `strategy` blocks, of course).